src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -2382,6 +2383,10 @@ private function addHtmlSanitizerSection(ArrayNodeDefinition $rootNode, callable
                                 ->fixXmlConfig('with_attribute_sanitizer')
                                 ->fixXmlConfig('without_attribute_sanitizer')
                                 ->children()
+                                    ->enumNode('default_action')
+                                        ->info('Defines how the sanitizer must behave by default.')
+                                        ->values(array_map(static fn (HtmlSanitizerAction $action): string => $action->value, HtmlSanitizerAction::cases()))
```

Comment:
```suggestion
                                        ->values(array_column(HtmlSanitizerAction::cases(), 'value'))
```

Reply:
but this won't work of the component is not installed, isn't it?

Reply:
Yes good point. Just came here to say https://github.com/symfony/symfony/pull/57686 might allow this to be cleaner, but indeed if the enum class is missing what do we do?